libethereum/Executive.cpp

```diff
@@ -459,6 +459,12 @@ bool Executive::go(OnOpFunc const& _onOp)
             m_excepted = toTransactionException(_e);
             revert();
         }
+        catch (InternalVMError const& _e)
+        {
+            cwarn << "Internal VM Error (" << *boost::get_error_info<errinfo_evmcStatusCode>(_e) << ")\n"
```

Comment:
The other exception messages here don't have an explicit new line character.

Reply:
It looks a bit better if the diagnostic_information is on the next lines.